cubedash/summary/_schema.py

```diff
@@ -170,6 +176,16 @@
     Column("has_region", Integer),
 )
 
+# The geometry of each unique 'region' for a product.
+REGION = Table(
+    "mv_region",
+    _REF_TABLE_METADATA,
+    Column("dataset_type_ref", SmallInteger, nullable=False),
```

Comment:
Not to lump in many changes. Shall we bump primary key data type ?

Reply:
The corresponding upstream ODC-core primary key that this links to is a SmallInt, so I'm hesitant to deviate from it. We just need to avoid making bad upsert statements again...